forge-gui/res/cardsfolder/upcoming/sidar_jabari_of_zhalfir.txt

```diff
@@ -0,0 +1,16 @@
+Name:Sidar Jabari of Zhalfir
+ManaCost:1 W U B
+Types:Legendary Creature Human Knight
+PT:4/3
+T:Mode$ AttackersDeclared | ValidAttackers$ Card.Knight+YouCtrl | TriggerZones$ Command | PresentZone$ Command | Execute$ TrigLoot | TriggerDescription$ Eminence — Whenever you attack with one or more Knights, if CARDNAME is in the command zone or on the battlefield, draw a card, then discard a card.
+T:Mode$ AttackersDeclared | ValidAttackers$ Card.Knight+YouCtrl | TriggerZones$ Battlefield | PresentZone$ Battlefield | Execute$ TrigLoot | Secondary$ True | TriggerDescription$ Eminence — Whenever you attack with one or more Knights, if CARDNAME is in the command zone or on the battlefield, draw a card, then discard a card.
+SVar:TrigLoot:DB$ Draw | Defined$ You | NumCards$ 1 | SubAbility$ DBDiscard
+SVar:DBDiscard:DB$ Discard | Defined$ You | NumCards$ 1 | Mode$ TgtChoose
+K:Flying
+K:First Strike
+T:Mode$ DamageDone | ValidSource$ Card.Self | ValidTarget$ Player | CombatDamage$ True | Execute$ TrigReturn | TriggerZones$ Battlefield | TriggerDescription$ Whenever NICKNAME deals combat damage to a player, return target Knight creature card from your graveyard to the battlefield.
```

Comment:
NICKNAME parsing will fail here, need to think on that

Reply:
I guess CARDNAME would do for now ?

Reply:
yea probably better